content/articles/2018-08-scales-1-0-0.Rmarkdown

```diff
@@ -0,0 +1,98 @@
+---
+title: "scales 1.0.0"
+author: Dana Seidel
+date: '2018-07-30'
+slug: scales-1-0-0
+description: >
+  scales 1.0.0 is now on CRAN.
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/3kzlCL3rj8A
+  author: Ricardo Gomez Angel
+tags: []
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+We're delighted to announce the release of [scales](https://scales.r-lib.org/) 1.0.0. 
+This is a major release with significant changes to the popular formatter
+functions and added transformations.
+See the [News](https://scales.r-lib.org/news/index.html) for a detailed list of changes.
+
+Install the latest version with:
+```{r, eval = FALSE}
+install.packages("scales")
+```
+
+# New features
+## Formatter changes
+
+Thanks to the help of [@larmarange](https://github.com/larmarange), scales has 
+now added a generic formatter function `number_format()` that powers the majority
+of the formatters in the package. 
+
+This change altered the arguments of most of the formatters: 
+`comma_format()`, `percent_format()`, `unit_format()`, `dollar_format()`,
+`ordinal_format()`, and `scientific_format()` all gained new arguments allowing 
```

Comment:
Maybe list what the new arguments are? (in which case you should probably break it into two sentences)